iot/iot-infinispan-device-registry/pom.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.enmasse</groupId>
+    <artifactId>iot-application-base</artifactId>
+    <version>0.27-SNAPSHOT</version>
+    <relativePath>../iot-application-base</relativePath>
+  </parent>
+
+  <artifactId>iot-infinspan-device-registry</artifactId>
+  
+  <properties>
+    <infinispan.version>9.4.6.Final</infinispan.version>
+  </properties>
+
+  <dependencies>
+        <dependency>
+            <groupId>org.eclipse.hono</groupId>
+            <artifactId>hono-service-device-registry</artifactId>
+            <version>${hono.version}</version>
```

Comment:
Use dependencyManagement for versions, i.e. remove version attributes for all dependencies.

Reply:
So all the version information should be in the pom file in the `iot` module ? or in the main pom file of enmasse ? Or I should use `<dependencyManagement>` in this pom file?

Reply:
Use the iot/pom.xml dependencyManagement to start with, although I think we should merge that with the root pom.xml eventually.

(CC @k-wall )